ts/src/test/test.ts

```diff
@@ -231,22 +231,24 @@ export default class testMainClass extends baseMainTestClass {
         await this.importFiles (exchange);
         assert (Object.keys (this.testFiles).length > 0, 'Test files were not loaded'); // ensure test files are found & filled
         this.expandSettings (exchange);
-        const symbol = this.checkIfSpecificTestIsChosen (symbolArgv);
+        const symbol = this.checkIfSpecificTestIsChosen (exchange, symbolArgv);
         await this.startTest (exchange, symbol);
         exitScript (0); // needed to be explicitly finished for WS tests
     }
 
-    checkIfSpecificTestIsChosen (symbolArgv) {
+    checkIfSpecificTestIsChosen (exchange, symbolArgv) {
         if (symbolArgv !== undefined) {
-            const testFileNames = Object.keys (this.testFiles);
+            const testConfig = config ();
+            const tests = exchange.deepExtend (testConfig['exchange'], testConfig[exchange.id]);
+            const testNames = Object.keys (tests);
             const possibleMethodNames = symbolArgv.split (','); // i.e. `test.ts binance fetchBalance,fetchDeposits`
             if (possibleMethodNames.length >= 1) {
-                for (let i = 0; i < testFileNames.length; i++) {
-                    const testFileName = testFileNames[i];
+                for (let i = 0; i < testNames.length; i++) {
+                    const testName = testNames[i];
                     for (let j = 0; j < possibleMethodNames.length; j++) {
                         const methodName = possibleMethodNames[j];
-                        if (testFileName === methodName) {
-                            this.onlySpecificTests.push (testFileName);
+                        if (testName.indexOf (methodName) >= 0) {
```

Comment:
can you remind what was the need to change from `===` comparison?

Reply:
Yes It's to include if we have several tests of the same function with different names. Like watchTickers spot, watchTickers with some param, etc..

Reply:
i'll propose another structure for that config-item tomorrow and lmk then your thoughts.